scripts/zephyr_module.py

```diff
@@ -480,7 +480,7 @@ def git_revision(path):
                 if rc:
                     return revision + '-dirty', True
                 return revision, False
-        return None, False
+        return "unknown", False
```

Comment:
I think "unknown" is fine, but want to mention that this can happen in different scenarios, like:
- git returning error, like `fatal: detected dubious ownership`
- Folder is not a git repo

I think "unknown" is fine in both cases, but one could argue that there is a difference between a non-git repo, and a git repo which fails the git command, and thus different strings should be used to reflect this.

Reply:
I could an argument either way.  Let me know if you have a preference and we can find a solution.

Reply:
As I wrote, then `unknown` is fine with me, and keeps the code simpler.

The reason for making the comment is, A) to make others aware that there are possibilities to differentiate, so they can comment if they have a preference. B) As a possible future reference in case topic is brought up again, then we know why this decision was made.

Reply:
I see, that all makes sense.  Thanks for walking me through my first zephyr PR!